glossary.html

```diff
@@ -9,35 +9,37 @@
 	<nav>
 		<ul>
 			<li>
-			<a href="">Home</a>
+			<a href="index.html">Home</a>
 			</li>
 			<li>
-			<a href="">Links</a>
+			<a href="links.html">Links</a>
 			</li>
 			<li>
-			<a href="">Glossary</a>
+			<a href="glossary.html">Glossary</a>
 			</li>
 		</ul>
 	</nav>
 </header>
 
 <main>
 	<h1>Acronym Glossary</h1>
-		<h2>HTML Basic</h2>
-		<p>HTML defines the structure and layout of a Web document by using a variety of tags and attributes.</p>
-		<h2>CSS basics</h2>
-		<p>CSS describes how HTML elements are to be displayed on screen, paper, or in other media</p>
-		<h2>internet Basic</h2>
-		<p>The Internet is essentially a global network of computing resources. You can think of the Internet as a physical collection of routers and circuits as a set of shared resources</p>
-		<h2>HTTP</h2>
-		<p>HTTP means HyperText Transfer Protocol. HTTP is the underlying protocol used by the World Wide Web and this protocol defines how messages are formatted and transmitted, and what actions Web servers and browsers should take in response to various commands</p>
+	<dl>
+		<dt>HTML Basic</dt>
+		<dd>HTML defines the structure and layout of a Web document by using a variety of tags and attributes.</dd>
+		<dt>CSS basics</dt>
+		<dd>CSS describes how HTML elements are to be displayed on screen, paper, or in other media</dd>
+		<dt>internet Basic</dt>
+		<dd>The Internet is essentially a global network of computing resources. You can think of the Internet as a physical collection of routers and circuits as a set of shared resources</dd>
+		<dt>HTTP</dt>
+		<dd>HTTP means HyperText Transfer Protocol. HTTP is the underlying protocol used by the World Wide Web and this protocol defines how messages are formatted and transmitted, and what actions Web servers and browsers should take in response to various commands</dd>
+	</dl>
```

Comment:
The lab asked you to use a 'definition list', which is what you see in my correction here